src/ConnectCore.js

```diff
@@ -87,6 +88,31 @@ class ConnectCore {
     // TODO throw error if this.network not part of network set in Credentials
     this.canSign = !!this.credentials.settings.signer && !!this.credentials.settings.address
     this.pushToken = null
+    this.PubSub = PubSub
+
+    this.loadPlugins()
+  }
+
+  loadPlugins() {
```

Comment:
It would be better to have this live in another class used exclusively for buttons, since it adds extra complexity to non button users.